packages/lib/src/ui/components/Image/Image.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import styled from "@emotion/styled"
+import NextImage from "next/image"
+import { Image as ImageType } from "../../../../src/types/data"
```

Comment:
Nefungovalo by to i takhle?
```suggestion
import { Image as ImageType } from "../../../types/data"
```